src/v5_hal/firmware/src/main.cpp

```diff
@@ -59,7 +60,9 @@ void initialize() {
 
 	tank_drive_node = new TankDriveNode(node_manager, "drivetrain", primary_controller, 
 		left_front_drive, left_rear_drive, right_front_drive, right_rear_drive);
```

Comment:
Remove this for now and just have the HolonomicDriveNode

Reply:
The problem with that is that OdometryNode, ProgrammingSkillsAuton, and AutonManagerNode all expect a tank drive so if we removed TankDriveNode we wouldn't be able to push to master

Reply:
ah yeah, that's true. We'll have to refactor those elsewhere, forgot about that. For now I'd probably comment out the x-drive node then (just so we don't have two subsystems sending commands to the same motors) with a comment to fix once the other subsystems are removed.

These changes should be fixed once I eventually get around to having a generic "drivetrain" type